src/main/java/io/confluent/connect/jdbc/TableMonitorThread.java

```diff
@@ -102,7 +104,7 @@ private boolean updateTables() {
     synchronized (db) {
       final List<String> tables;
       try {
-        tables = JdbcUtils.getTables(db);
+        tables = JdbcUtils.getTables(db,table_types);
```

Comment:
I can only see a case of people wanting all tables and maybe some whitelisted/blacklisted views. Maybe just disallow an empty list and default to Tables if specified?